apps/src/studio/studio.js

```diff
@@ -2660,6 +2752,36 @@ Studio.displayScore = function() {
   score.setAttribute('visibility', 'visible');
 };
 
+Studio.displayFloatingScore = function(changeValue) {
+  if (!level.floatingScore) {
+    return;
+  }
+
+  var sprite = Studio.sprite[0];
+  var floatingScore = document.getElementById('floatingScore');
+  floatingScore.textContent = changeValue > 0 ? ("+" + changeValue) : changeValue;
+  floatingScore.setAttribute('x', sprite.x + sprite.width/2);
+  floatingScore.setAttribute('y', sprite.y + sprite.height/2);
+  floatingScore.setAttribute('opacity', 1);
+  floatingScore.setAttribute('visibility', 'visible');  
+}
+
+Studio.updateFloatingScore = function() {
+  if (!level.floatingScore) {
+    return;
+  }
+
+  var floatingScore = document.getElementById('floatingScore');
+  var y = floatingScore.getAttribute('y');
+  var opacity = floatingScore.getAttribute('opacity');
+  if (opacity > 0) {
+    opacity -= 0.025;
```

Comment:
Can you define constants for the opacity change of 0.025 and the y change of -2? That will make it more obvious for others who read this later. You can place them in constants.js.